rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
```

Comment:
- [ ]  It will be great to create many to many relationships between a book and a person through rentals. This is a major requirement for this project. Since this is just pure ruby, kindly don't use rails associations to create relationships instead use methods and variables. To read more about this kindly follow this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)